File: app/src/main/java/com/sample/android/elm/login/view/LoginActivity.java
package com.sample.android.elm.login.view;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import com.androidjacoco.sample.R;

public class LoginActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.login_main);

        if (getSupportFragmentManager().findFragmentByTag("TAG") == null) {
            getSupportFragmentManager().beginTransaction()
                    .replace(R.id.login_fragment, new LoginFragment(), "TAG")
                    .commit();
        }

    }

}

File: app/src/main/java/com/sample/android/elm/login/view/LoginActivity.kt
package com.sample.android.elm.login.view

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import com.androidjacoco.sample.R

class LoginActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.login_main)

        if (supportFragmentManager.findFragmentByTag("TAG") == null) {
            supportFragmentManager.beginTransaction()
                .replace(R.id.login_fragment, LoginFragment(), "TAG")
                .commit()
        }

    }

}
